protos/sources/vega/markets.proto

```diff
@@ -241,6 +241,8 @@ message Market {
     STATE_TRADING_TERMINATED = 8;
     // Settlement triggered and completed as defined by product
     STATE_SETTLED = 9;
+    // Market has been suspended via governance
+    STATE_SUSPENDED_VIA_GOVERNANCE = 10;
```

Comment:
Can't we reuse `STATE_SUSPENDED` and use the auction trigger `AUCTION_TRIGGER_GOVERNANCE_SUSPENSION = 7;` distinguish between the reason?

Reply:
technically we could, but why?  I'd prefer it to be explicit unless there's any benefit for reusing the existing state_suspended.

Reply:
Can we at least change the trading mode one to `TRADING_MODE_SUSPENDED_VIA_GOVERNANCE`? I just find it odd that we can have a market state of `STATE_SUSPENDED` and a trading mode of `TRADING_MODE_SUSPENDED`, but a market state of `SUSPENDED` =/=> a trading mode of `SUSPENDED`.

Reply:
sure